### PR TITLE
[MNOE-1088] Quote Based Pricing

### PR DESCRIPTION
**Pull request by paulstuartparker**

### CODE REVIEW

File: api/config/routes.rb [hunk 1]
```diff
@@ -152,7 +152,9 @@
 
         resources :audit_events, only: [:index]
 
+
```
Comment (adamaziz15) [inline api/config/routes.rb]: remove extra newline

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/subscriptions_controller.rb [hunk 2]
```diff
@@ -21,7 +21,6 @@ def show
   # POST /mnoe/jpi/v1/organizations/1/subscriptions
   def create
     authorize! :manage_app_instances, parent_organization
-
```
Comment (adamaziz15) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/subscriptions_controller.rb]: add this newline back in for readability :)

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/subscriptions_controller.rb [hunk 3]
```diff
@@ -35,7 +34,6 @@ def create
       subscription.relationships.product_contract = MnoEnterprise::ProductContract.new(id: params[:subscription][:product_contract_id])
     end
     subscription.save!
-
```
Comment (adamaziz15) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/subscriptions_controller.rb]: this one as well

File: core/lib/mno_enterprise/concerns/models/organization.rb [hunk 4]
```diff
@@ -11,7 +11,6 @@ module MnoEnterprise::Concerns::Models::Organization
     custom_endpoint :trigger_app_instances_sync, on: :member, request_method: :post
     custom_endpoint :freeze, on: :member, request_method: :patch
     custom_endpoint :unfreeze, on: :member, request_method: :patch
-
```
Comment (adamaziz15) [inline core/lib/mno_enterprise/concerns/models/organization.rb]: annnd this one

File: api/config/routes.rb [hunk 5]
```diff
@@ -155,6 +155,7 @@
         resources :audit_events, only: [:index]
 
         if Settings&.dashboard&.marketplace&.provisioning
+          resources :quotes, only: [:index, :show, :create]
```
Comment (adamaziz15) [inline api/config/routes.rb]: We only need `:create` for this one now right?

File: core/lib/mno_enterprise/concerns/models/organization.rb (hunk 4 above)
Comment (ouranos) [reply core/lib/mno_enterprise/concerns/models/organization.rb]: This wasn't fixed

**Pull request merged**